src/Core/AdminConsole/Services/Implementations/OrganizationService.cs

```diff
@@ -908,6 +912,8 @@ public async Task<IEnumerable<Tuple<OrganizationUser, string>>> ResendInvitesAsy
         IEnumerable<Guid> organizationUsersId)
     {
         var orgUsers = await _organizationUserRepository.GetManyAsync(organizationUsersId);
+        _userInviteDebuggingLogger.Log(orgUsers);
```

Comment:
The logs in this file should account for all reinvite flows except for provider users and emergency access.